session-replay/Tests/DatadogSessionReplayTests/Mocks/FoundationMocks.swift

```diff
@@ -35,3 +35,23 @@ extension Date: AnyMockable, RandomMockable {
         return dateComponents.date!
     }
 }
+
+extension FixedWidthInteger where Self: RandomMockable {
```

Comment:
@maxep most of these `FoundationoMocks` are copy & paste from RUM SDK's test target. There's a vast opportunity for code reuse if we move it to separate "test helper" module. Something to track in V2, WDYT?